### PR TITLE
Category of elements as a wild functor to CAT.

### PR DESCRIPTION
**Pull request by anuyts**

### CODE REVIEW

File: Cubical/Categories/Category.agda [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
-  Precategory          Type   Type   No
+  Wild category        Type   Type   No
   Category             Type   Set    No
   Univalent Category   Type   Set    Yes
 
   The most useful notion is Category and the library is hence based on
   them. If one needs precategories then they can be found in
-  Cubical.Categories.Category.Precategory
+  Cubical.WildCat (so it's not considered part of the Categories sublibrary!)
```
Comment (maxsnew) [inline Cubical/Categories/Category.agda:14]: Good catch here. There is a mention of precategories on line 13 that should also be changed to "wild categories"

**Review by maxsnew — CHANGES_REQUESTED** [review]
Code looks good, just fix that comment and I'll merge

**Review by maxsnew — APPROVED** [review]

**Comment by maxsnew** [comment]
@anuyts why did you mark this as a draft?

**Comment by anuyts** [comment]
…because I was in the process of cleaning up my equality proofs.

**Pull request merged**